notebooks/203-meter-reader/203-meter-reader-digital.ipynb

```diff
@@ -0,0 +1,1085 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "29b161d0-8144-4b3f-88fe-deb6abd21f1c",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Meter Reader Digital\n",
+    "This notebook shows how to create a meter reader with OpenVINO Runtime. We use the pre-trained [PP-OCR](https://github.com/PaddlePaddle/PaddleOCR) to build up a inference task pipeline:\n",
+    "\n",
+    "1. Configure the screen area of the meter reader.\n",
+    "2. Config the layout information of the meter reader.\n",
+    "3. Pre-process the image based on the given information.\n",
+    "4. Perform OCR recognition.\n",
+    "5. Structure output information.\n",
+    "\n",
+    "In this notebook, we recognize two types of meters. The biggest difference between the two cases is that the display screens of the two devices are different. Case 1 is a field strength meter, the screen in the meter is not Segment code LCD, which means that we can read the pure text on the screen. Case 2 is a household electricity meter, the screen of the meter is Segment code LCD, which means we cannot only see the text but also the outline of the text in the background. The background outline can also be mistaken by the model as content that needs to be recognized, so we need to add some preprocessing to eliminate the impact of background information.\n",
```

Comment:
```suggestion
    "This notebook will demonstrate how to read meters with different screens. In case 1 there is is a field strength meter with  general LCD screen, which means that we can read the pure text from the screen. In case 2, there is a household electricity meter with segment code LCD screen, which means we can only see the text but also the outline of the text in the background. The background outline can also be misread by the model as content that needs to be recognized, so we need to add some preprocessing module to eliminate the background information.\n",
```